packages/data-point/lib/reducer-types/reducer-list/resolve.test.js

```diff
@@ -80,19 +80,20 @@ describe('resolve#reducer.resolve - reducer model', () => {
     ).then(result => expect(result.value).toEqual(testData))
   })
 
-  test('multiple models', () => {
+  test('multiple models for reducer model', () => {
```

Comment:
I think this can be rewritten, but I don't know it well enough to suggest an alternative. Sorry. 😞